meeting-notes/2019-06-20-notes.md

```diff
@@ -0,0 +1,172 @@
+
+* Right now the checks are only "technical checks". But there is not discussion of usability in the review template. 
+    * does it align with common practices
+    * there is more space then for additional feedback then...
+
+
+* Actional feedback in the review. right now it's a HUGE issue. (so much text). But we didn't open issues in this particular example. 
+
+* Resolution: 
+    * Reboot this review and potentially update the template in this issue. 
+    * Do we want to start fresh given lessons learned??
+
+* If we are encouraging reviewers to open issues - could they also open PR's ? That would be a nice way to help the author along in their improvements to the package. 
+
+* Make this an opt out?? opt out of having issues and pr's in your repo. we could have it checked by default an they can open out by unchecking it.  
+    * update the checkbox to allow for people opening issues and pr's 
+
+3. [Earthpy](https://github.com/pyOpenSci/software-review/issues/3) - need reviewers?
+    If the list below works, Luiz can update the issue and we can start again! If that is happening i may update the version as we just bumped it!! 
+    * leo
+    * Martin (FYI, I am a bio (not geo) person😃)
+    * > [name=Leah Wasser]i think that is fine!
+        * Luiz: that's sort of on purpose, you could focus more on package org and docs, and leo on the geo parts =]
+        
+4. Nbless -- Status: need a second reviewer if leo does earthpy
+    * Filipe (Thank you for completing this review!)
+    * Second reviewer wanted: who is interested? kylen perhaps?
+        * [Kylen volunteered](https://github.com/pyOpenSci/software-review/issues/7#issuecomment-504101498) 
+    * feedback (if there is time). We implemented a process that included opening issues in the package repo and that seemed to work great!
+
+5. Open Issues
+How should we handle this todo list? maybe a space in our discourse forum that gets referenced in our checkins?? This gets into task management which i think we can do in github but i have always used asana :) 
+Open Issues 
+- [ ] https://github.com/pyOpenSci/dev_guide/issues/17
+    - [ ] sign up here
+- [ ] https://github.com/pyOpenSci/dev_guide/issues/27
+- [ ] https://github.com/pyOpenSci/dev_guide/issues/28
+
+4. Plan to discuss gitter for pyOpenSci in future meeting
+
+5. Roles reminder (we can skip this if there's not time today)
+
+## Role Definition Revisited (please feel free to edit this or leave comments!!)
+
+Starting with the [rOpenSci structure](https://ropensci.org/about/) but we may want to adjust accordingly. And ofcourse the time commitment can really be flexible but just trying to give folks a sense of time that might be associated with each role. 
+
+## Leadership (3-5): 
+* Responsible for making final decisions about the direction of POS
+* POS
+* Communication: weekly / bi weekly, slack, discuss private forum (Chris H can help us set this up i think??, emails)
```

Comment:
lemme know if you'd like me to do something here

Reply:
oh - perhaps what would be good is just telling people about the various forums in discourse as you did set this up. we aren't using it yet however.